packages/fifteen-logic/package.json

```diff
@@ -1,27 +1,20 @@
 {
   "name": "fifteen-logic",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
```

Comment:
@sappharx I added a description but main isn't being leveraged so I yanked it. Thoughts/improvements?